### PR TITLE
filecloud: find string not found for a file

### PR DESCRIPTION
**Pull request by zodman**
on filecloud.io

### CODE REVIEW

**Comment by zodman** [comment]
fixing it i filecloud.io use recaptcha to show the file ..

File: filecloud.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Plowshare filecloud.io module
+
```
Comment (mcrapet) [inline filecloud.sh]: Could you delete this line ?

File: mediafire.sh [hunk 2]
```diff
@@ -225,7 +225,7 @@ mediafire_download() {
 
     # Only get site headers first to capture direct download links
     URL=$(curl --head "$BASE_URL/?$FILE_ID" | grep_http_header_location_quiet) || return
-
+    log_debug $URL
```
Comment (mcrapet) [inline mediafire.sh]: Delete this debug line too.

**Comment by zodman** [comment]
changes apply!

**Comment by mcrapet** [comment]
Applyed, then.
Next time, rebase it to have 1 single commit.
One PR per module.

**Pull request merged**